src/Microsoft.ML.Featurizers/TimeSeriesImputer.cs

```diff
@@ -83,6 +83,16 @@ public static class TimeSeriesImputerExtensionClass
     /// | Output column data type | All Types |
     /// | Exportable to ONNX | No |
     ///
+    /// The TimeSeriesImputer imputes missing rows and column data per grain (category), based on the dates in the date column. This operation needs to happen to every column in the IDataView,
```

Comment:
Restructure summary and remarks so that the summary is just that - a one line description of what the class and when you would use it (if appropriate). Have a look at this page to see what the descriptions look like once they are rendered as HTML: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.featurizers?view=ml-dotnet-preview-latest&branch=smoke-test-preview

Put the rest of the information into the remarks section. This is what a developer will look at when they are using the class rather than comparing transforms to see which one they need to use: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.featurizers.timeseriesimputerestimator?view=ml-dotnet-preview-latest&branch=smoke-test-preview

Reply:
Left the first line in the summary section as its a pretty good summary. Moved the rest down into the remarks section.